RationaleMCP/0031/annotations.md

```diff
@@ -36,6 +36,47 @@ These are all the non-vendor specific annotations inherited from full Modelica t
 - `hideResult` — Applied to a parameter or variable, implies the variable should not be included in the simulation output
 - `choices` — Applied to a parameter or variable, can be used to enumerate and tag different values for parameter input
 
+These are the new annotations introduced in Flat Modelica, each explained in more detail below:
+- [`protected`](#protected) — Indicate whether component declaration comes from protected section in original full Modelica model
```

Comment:
Web Meeting: Hans, Henrik, Gerd, Martin, Oliver

Gerd: Using a Modelica keyword for the annotations doesn't feel right.

Candidate names:

- Protected
- isProtected